.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Great job adding the file `test.md` :clap: but please list it in the `.gitignore` file since it is one of the requirements :+1: